internal/services/cache_manufacturer_service_test.go

```diff
@@ -0,0 +1,77 @@
+package services
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/DIMO-Network/identity-api/internal/config"
+	test "github.com/DIMO-Network/identity-api/internal/helpers"
+	"github.com/DIMO-Network/identity-api/models"
+	"github.com/DIMO-Network/shared/db"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/suite"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/volatiletech/sqlboiler/v4/boil"
+)
+
+type ManufacturerCacheServiceTestSuite struct {
+	suite.Suite
+	ctx       context.Context
+	pdb       db.Store
+	container testcontainers.Container
+	settings  config.Settings
+	logger    zerolog.Logger
+}
+
+func (o *ManufacturerCacheServiceTestSuite) SetupSuite() {
+	o.ctx = context.Background()
+	o.pdb, o.container = test.StartContainerDatabase(o.ctx, o.T(), "../../migrations")
+
+	o.settings = config.Settings{}
+
+	o.logger = zerolog.New(os.Stdout).With().Timestamp().Str("app", test.DBSettings.Name).Logger()
+}
+
+// TearDownTest after each test truncate tables
+func (s *ManufacturerCacheServiceTestSuite) TearDownTest() {
+	test.TruncateTables(s.pdb.DBS().Writer.DB, s.T())
+}
+
+// TearDownSuite cleanup at end by terminating container
+func (o *ManufacturerCacheServiceTestSuite) TearDownSuite() {
+	fmt.Printf("shutting down postgres at with session: %s \n", o.container.SessionID())
+
+	if err := o.container.Terminate(o.ctx); err != nil {
+		o.T().Fatal(err)
+	}
+}
+
+// Test Runner
+func TestManufacturerCacheServiceTestSuite(t *testing.T) {
+	suite.Run(t, new(ManufacturerCacheServiceTestSuite))
+}
+
+func (o *ManufacturerCacheServiceTestSuite) Test_Manufacturer_All_Success() {
```

Comment:
Two tests that I see missing here 

- Since you want to return an error when manufacturers are not found, can you write a test for that scenario too.
- Can we verify the cache data is actually being returned early as implemented [here](https://github.com/DIMO-Network/identity-api/pull/88/files#diff-c76fc5f9f192673157bcdbfeda344b15ec590ab9351baa73bc8fe209c857a8d9R44)